tests/manager/scripts/test_playtime_entries.py

```diff
@@ -576,11 +576,17 @@ def test_do_run(
         # We're using the RecursiveEquivalencyCache, so must refresh it.
         EquivalentIdentifiersCoverageProvider(db.session).run()
 
+        # one month + 3 days ago : in scope
         playtime(
             db.session, identifier, collection, library, dt1m(3), 1, loan_identifier
         )
+        # one month + 4 days ago : in scope  should be added
         playtime(
-            db.session, identifier, collection, library, dt1m(31), 2, loan_identifier
+            db.session, identifier, collection, library, dt1m(4), 2, loan_identifier
+        )
+        # one month + 31 days ago : out of range of reporting period
```

Comment:
Minor quibble about this comment: it sounds like it's over two months ago. Could we re-word to something like:

```
# out of range: after end of default reporting period
```

But, good to have this labeled, so that we know what's up next time! For future us! 🥂